### PR TITLE
Update README to mention JRuby and fix circular require warning

### PR DESCRIPTION
**Pull request by ivoanjo**
This PR:
* updates the README to always recommend `pry-byebug` for MRI users (as you should know better if you're still on < 1.9)
* cleans up a circular require warning
* bumps the gem version to 0.2.5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.2.5 (2016-08-26)
```
Comment (nixme) [inline CHANGELOG.md]: I usually do the release as it's own commit so that the changes are their own commit and easier to cherry-pick into other branches, etc.

so this would be:

```markdown
## UNRELEASED
```

in this PR. And then you can just create a new commit directly to master updating the version number and changelog here and then tagging and pushing to rubygems.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
-### Using MRI 1.9.2+? Switch to [**pry-debugger**][pry-debugger].
-### Using MRI 2+? Switch to [**pry-byebug**][pry-byebug].
+# pry-nav
 
-Same features as **pry-nav** but with faster tracing, breakpoints, and more.
+_A simple execution control add-on for [Pry][pry]._
 
-* * *
+Using MRI? We recommend [`pry-byebug`][pry-byebug] instead!
```
Comment (nixme) [inline README.md]: I'm okay with changing the wording, and removing the reference to `pry-debugger`, but would you mind keeping this at the top. Making it bolder and keeping at the top helps most users— lots of old pry guides still mention pry-nav, and most users are served better by pry-byebug.

File: lib/pry-nav/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module PryNav
-  VERSION = '0.2.4'
+  VERSION = '0.2.5'
```
Comment (nixme) [inline lib/pry-nav/version.rb]: see note above about `VERSION`

**Comment by nixme** [comment]
Also, can we close #19 if your changes to jRuby mean it's fixed?

File: README.md (hunk 2 above)
Comment (ivoanjo) [inline README.md]: Agree, changed it back on 3fd0650bfb761e2d4ea98d879e076d6a2053a1d9

File: lib/pry-nav/version.rb (hunk 3 above)
Comment (ivoanjo) [inline lib/pry-nav/version.rb]: Sounds good, I've removed the commit with the version bump

File: CHANGELOG.md (hunk 1 above)
Comment (ivoanjo) [inline CHANGELOG.md]: I've amended a10cd4808088a1eb3e4b5c30ce7cb4a2e0c14db2 to contain the changelog entry as you've suggested.

**Comment by ivoanjo** [comment]
Yeah, I think #19 can be closed too. The `-Xcext.enabled=true` option also being discussed there I'm not sure it was ever needed but I can confirm that it's not needed right now, nor really supported in JRuby 9k.

**Comment by jasonrclark** [comment]
Any chance of a gem release with this change in it? I'm using JRuby, so can't do `pry-byebug` and I'm trying to run tests with warnings on, but this spams the output a lot. Thanks!

**Comment by ivoanjo** [comment]
@jasonrclark unfortunately the author has been quite irresponsive to my communication attempts and other PRs.

My plan was to push improvements upstream but I ended up working on a fork of @nixme's other debugging gem: https://github.com/ivoanjo/pry-debugger-jruby .
